pkg/api/helpers.go

```diff
@@ -52,3 +54,19 @@ func GetFieldLabelConversionFunc(supportedLabels map[string]string, overrideLabe
 		return "", "", fmt.Errorf("field label not supported: %s", label)
 	}
 }
+
+// TODO: Move this upstream
+func ExportObjectMeta(objMeta *kapi.ObjectMeta, exact bool) {
```

Comment:
> I don't like this method modifies object in place. I'd rather prefer this returns modified object.

Modifying in place is the whole pattern of `RESTExportStrategy`.  It would be a little strange to not do it here.